### PR TITLE
Fix sampling algorithm

### LINKED ISSUE #64: Using zipkin.tracing_percent > 50.0 results in 100% sampling
This is the same issue as https://github.com/Yelp/py_zipkin/issues/8

The problem is that `int((1.0 / x) * 100)` is always 1 for x > 50.

### PR DESCRIPTION
**Pull request by kaisen**
Fixes #64

I also renamed webtest's TestApp to WebTestApp so we don't get anymore pytest warnings.

### CODE REVIEW

File: pyramid_zipkin/request_helper.py [hunk 1]
```diff
@@ -79,18 +80,16 @@ def should_not_sample_route(request):
...
+def should_sample_as_per_zipkin_tracing_percent(tracing_percent):
     """Calculate whether the request should be traced as per tracing percent.
 
     :param tracing_percent: value between 0.0 to 100.0
     :type tracing_percent: float
-    :param req_id: unique request id of the request
     :returns: boolean whether current request should be sampled.
     """
     if tracing_percent == 0.0:  # Prevent the ZeroDivision
         return False
```
Comment (bplotnick) [inline pyramid_zipkin/request_helper.py]: You don't need this anymore

File: pyramid_zipkin/request_helper.py [hunk 2]
```diff
@@ -79,18 +80,16 @@ def should_not_sample_route(request):
...
-def should_sample_as_per_zipkin_tracing_percent(tracing_percent, req_id):
+def should_sample_as_per_zipkin_tracing_percent(tracing_percent):
     """Calculate whether the request should be traced as per tracing percent.
 
     :param tracing_percent: value between 0.0 to 100.0
     :type tracing_percent: float
-    :param req_id: unique request id of the request
     :returns: boolean whether current request should be sampled.
     """
     if tracing_percent == 0.0:  # Prevent the ZeroDivision
```
Comment (mjbryant) [inline pyramid_zipkin/request_helper.py]: This no longer prevents ZeroDivision. You can get rid of this if you want, or you can just remove the comment. It's probably _marginally_ faster if tracing_percent == 0.0.

File: tests/acceptance/server_span_test.py [hunk 3]
```diff
@@ -61,33 +49,33 @@ def test_unsampled_request_has_no_span(thrift_obj, default_trace_id_generator):
...
 
-    TestApp(main({}, **settings)).get('/sample', status=200)
+    WebTestApp(main({}, **settings)).get('/sample', status=200)
 
     assert thrift_obj.call_count == 0
 
 
 @mock.patch('py_zipkin.logging_helper.thrift_obj_in_bytes', autospec=True)
-def test_blacklisted_route_has_no_span(thrift_obj, sampled_trace_id_generator):
+def test_blacklisted_route_has_no_span(thrift_obj, default_trace_id_generator):
```
Comment (mjbryant) [inline tests/acceptance/server_span_test.py]: I think you can/should delete the sampled_trace_id_generator now. That just generated a trace ID that would always be sampled, and that's not how we do sampling decisions any more after this branch.

**Comment by mjbryant** [comment]
👍

**Comment by bplotnick** [comment]
:shipit:

**Pull request merged**